evap/evaluation/templates/faq.html

```diff
@@ -33,19 +33,20 @@ <h4 id="{{ section.id }}-s">{{ section.title }}</h4>
 
 {% block additional_javascript %}
     <script type="text/javascript">
-        var anchor = window.location.hash.replace("#", "").split('-');
-        id = anchor[0];
-        type = 'q';
-        if(anchor.length > 1)
-            type = anchor[1];
+        const anchor = window.location.hash.replace("#", "").split('-');
+        const id = anchor[0];
+        const type = anchor[1];
 
         if(type == "q" && id != ""){
-            $("#collapse-" + id).collapse('show');
+            const collapse = $("#collapse-" + id);
```

Comment:
with the div renamed as suggested above, maybe call this `answer_div` or similar?